packages/site/.env

```diff
@@ -4,4 +4,5 @@
 # ... env.local - to override these env vars when running a LOCALHOST version
 # ... env.test  - to override these env vars when running a TEST version
 # ... env.prod  - to override these env vars when running a PROD version
-NEXT_PUBLIC_ENABLE_LOGIN=false
\ No newline at end of file
+NEXT_PUBLIC_ENABLE_LOGIN=false
+SITE_URL=https://mosaic-mosaic-dev-team.vercel.app/
```

Comment:
Vercel has an env variable for the site URL https://vercel.com/docs/concepts/projects/environment-variables/system-environment-variables#framework-environment-variables

I guess this wouldnt work for us at the moment because we generate the snapshot locally then upload it to vercel